src/dtoannotations/Validator.java

```diff
@@ -0,0 +1,70 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Validator {
+
+    public List<String> validate(Object object) throws IllegalAccessException {
+        List<String> list = new ArrayList<>();
+        Class<?> objectClass = object.getClass();
+        Field[] declaredFields = objectClass.getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            Object fieldValue = declaredField.get(objectClass);
+            if (declaredField.isAnnotationPresent(Length.class)) {
+                Length length = declaredField.getAnnotation(Length.class);
+                if (fieldValue.toString() != null) {
+                    if (fieldValue.toString().length() < length.min() || fieldValue.toString().length() > length.max()) {
+                        list.add("\"" + declaredField.getName() + "\" " +
+                                "Field has an error, it doesn't match the required length. It is - " + fieldValue);
+                    }
+                }
+            }
+            if (declaredField.isAnnotationPresent(Email.class)) {
+                List<Character> regex = new ArrayList<>();
```

Comment:
Change email validation using **REGEX**
Helper [URL](https://mkyong.com/regular-expressions/how-to-validate-email-address-with-regular-expression/)

Reply:
done